src/pages/Shipping/index.js

```diff
@@ -3,4 +3,5 @@ import React from "react";
 const Shipping = () => {
   return <div>Shipping Page in Progress</div>;
 };
+
```

Comment:
Remove this file from a commit.

Reply:
@Pankaj-Sati  remove this file from a commit. It will create conflict.

Reply:
Resolved